CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.5.4dev
 
+* [Fix] Shows the community link to the exception message of ValueError and TypeError
```

Comment:
[Fix] Adds community link to `ValueError` and `TypeError` (#30)

Reply:
You know how to write the better change log 👍 
Fix in [6a3b45d](https://github.com/ploomber/jupysql/pull/106/commits/6a3b45d7aca8a3df9f6971f682fe2307e9442155)